patches/buildroot/0001-disable-shared-for-host-builds-of-gmp-isl-mpc-mpfr.patch

```diff
@@ -1,7 +1,7 @@
 From ce9da3957e01aa81af49210bb7fec3d8cb440690 Mon Sep 17 00:00:00 2001
 From: Ben Cressey <bcressey@amazon.com>
 Date: Sun, 6 Aug 2017 23:15:02 +0000
-Subject: [PATCH 1/6] disable shared for host builds of gmp, isl, mpc, mpfr
+Subject: [PATCH 1/5] disable shared for host builds of gmp, isl, mpc, mpfr
```

Comment:
I'm not too familiar with patching. I would think if I change anything (even if its just the number of patches) I'd have to change the commit SHAs right?